pipeline/tasks/gapic_tasks.py

```diff
@@ -82,9 +82,10 @@ class GapicCodeGenTask(task_base.TaskBase):
     default_provides = 'gapic_code_dir'
 
     def execute(self, language, toolkit_path, descriptor_set, service_yaml,
-                gapic_api_yaml, gapic_language_yaml, output_dir, api_name):
+                gapic_api_yaml, gapic_language_yaml, output_dir, short_name,
+                version):
         code_root = os.path.join(output_dir,
-                                 api_name + '-gapic-gen-' + language)
+                                 short_name + '-gapic-gen-' + language)
```

Comment:
Removing the version out of this path means that you can't build multiple major versions of the same API in one run.